packages/bot-web-ui/src/assets/sass/journal.scss

```diff
@@ -17,7 +18,6 @@
             align-items: center;
             -webkit-box-pack: center;
             width: 345px;
```

Comment:
This width value is giving the `There are no messages to display` component an unnecessary right margin in small screens. I think we can remove it?